pkg/cmd/server/start/start_master.go

```diff
@@ -558,6 +558,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunEndpointController()
 		kc.RunNamespaceController()
 		kc.RunPersistentVolumeClaimBinder()
+		kc.RunPersistentVolumeProvisioner(pvKClient)
```

Comment:
> @liggitt @deads2k - are we going to want separate identities for the provisioner and recycler?

Yeah, I think they'll end up with different identities.  I would expect more power for the provisioner.